transcription/transcription.go

```diff
@@ -29,3 +32,25 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// Transcribe a given file using a Sphinx jar. The fileName should be in
+// "name.wav" format and the jarName should be in "name.jar" format.
+func startTranscription(fileName string, jarName string) error {
+
+	cmd := exec.Command("java", "-jar", jarName, fileName)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return err
+	}
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+	//Currently copying output to Stdout
```

Comment:
Please add space between // and comment.

Is it valid to copy output to Stdout? Also, good to note where you plan to copy output to in future.